src/foam/java/refinements.js

```diff
@@ -288,6 +288,7 @@ foam.CLASS({
         name: this.name,
         type: this.javaReturns || 'void',
         visibility: 'public',
+        static : this.static ? 'static ' : '',
```

Comment:
This can be simplified to just:
static: this.static

since both statics are booleans.
Better, add a method to AbstractMethod called isStatic() which always returns false and add same method to Static that returns true. Then we don't need to store a static property in Static's, which doesn't really makes sense since since we don't want to let users make static Methods or non-static Statics.